lib/grape-swagger/doc_methods/path_string.rb

```diff
@@ -2,7 +2,7 @@ module GrapeSwagger
   module DocMethods
     class PathString
       class << self
-        def build(path, options = {})
+        def build(path, route_version, options = {})
```

Comment:
Maybe it'd be cleaner if this received the entire route? So `def build(route, options = {})`?